charts/drogue-cloud-examples/values.yaml

```diff
@@ -53,3 +53,19 @@ timescale:
     image: docker.io/library/postgres:14
   pusher:
     image: ghcr.io/drogue-iot/postgresql-pusher:0.1.0
+
+pusher:
+  enabled: true
+  knative: false
+  ingress: false
+  image: ghcr.io/drogue-iot/postgresql-pusher:0.1.0
+
+source:
+  kafka:
+    enabled: false
+  drogue:
+    enabled: true
+    image: ghcr.io/drogue-iot/drogue-event-source:0.1.0
+    endpoint: wss://ws-integration.sandbox.drogue.cloud
+    token: XXX
```

Comment:
We guess we need a way to provision this. Otherwise the example will break!?

Reply:
I would assume you would pass it to the helm command like `--set source.drogue.token=yyy`. Do you have something better in mind? BTW. Not providing a token worked for me on sandbox with public temperature app. I assumed it was "by design", but if that's not the case we should investigate it.

Reply:
No, that is "by configuration" :) … the `drogue-public-temperature` stream is readable by anonymous.

I guess we need a better story for the "examples" deployment. If we are thinking to deploy this standalone, then I think pre-configuring this with the sandbox is just fine. And not using a token at all, or requiring the user to pass in a token too.

When targeting the example deployment that comes with the "drgadm" script, we should create some "out the box" experience. Maybe we can use a "post install" job too, to auto-create an application, a token, and wire everything up automatically.

Reply:
> No, that is "by configuration" :) … the `drogue-public-temperature` stream is readable by anonymous.

That's what I meant and it's great for this easy getting started and showing what demos look like when there's actual devices and data already.

> I guess we need a better story for the "examples" deployment. If we are thinking to deploy this standalone, then I think pre-configuring this with the sandbox is just fine. And not using a token at all, or requiring the user to pass in a token too.
> 
> When targeting the example deployment that comes with the "drgadm" script, we should create some "out the box" experience. Maybe we can use a "post install" job too, to auto-create an application, a token, and wire everything up automatically.

So, let's aim to provide drogue event source configuration from the secret, and create a reusable job that connects to the cloud create resources and sets the values for the secrets? I would tackle this in a separate PR as a further improvement.

Reply:
This is now fixed with https://github.com/drogue-iot/drogue-cloud/pull/169